agent/iskakalunan.py

```diff
@@ -66,7 +66,7 @@ def run(self):
             subprocess.call(['python', 'runtest.py', '-h', '>', 'result'])
             return
 
-        subprocess.call(['git', 'checkout', 'master'])
+        subprocess.call(['git', 'checkout', 'dev'])
```

Comment:
I am thinking of reading a configuration file so that we don't need to commit code to github for switching branches. In that way, we can even switch to do different branches easily. However, it should be fine for now.